cmd/metacache-server-pool.go

```diff
@@ -318,58 +318,32 @@ func (es *erasureSingle) listPath(ctx context.Context, o *listPathOptions) (entr
 	//
 	// If we don't have a list id we must ask the server if it has a cache or create a new.
 	if o.ID != "" && !o.Transient {
-		// Create or ping with handout...
-		rpc := globalNotificationSys.restClientFromHash(pathJoin(o.Bucket, o.Prefix))
```

Comment:
I am not sure why this is removed, this means we are not going to use list caching anymore?

Reply:
> I am not sure why this is removed, this means we are not going to use list caching anymore?

Never mind, just found out this is erasure single